openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -63,6 +68,16 @@ def to_representation(self, user):
         :return: Dict serialized account
         """
         profile = user.profile
+        badges = settings.FEATURES.get('ENABLE_OPENBADGES') or False
+        if 'badges' in self.excluded_fields:
+            # Avoid doing this potentially expensive query-- it will be filtered out anyway.
```

Comment:
If the badges field is a potentially expensive query then why is it opt-out, instead of opt-in?  That is, why not have this be an "additional_fields" parameter instead of an "excluded_fields"?

Also, do you plan to paginate/cap this in a subsequent PR in case a user has a large number of badges?